src/vm/dllimport.h

```diff
@@ -226,8 +226,8 @@ inline bool SF_IsArrayOpStub           (DWORD dwStubFlags) { LIMITED_METHOD_CONT
 inline bool SF_IsMulticastDelegateStub  (DWORD dwStubFlags) { LIMITED_METHOD_CONTRACT; return (dwStubFlags == ILSTUB_MULTICASTDELEGATE_INVOKE); }
 #endif
 
-#ifdef FEATURE_STUBS_AS_IL
 inline bool SF_IsSecureDelegateStub  (DWORD dwStubFlags)    { LIMITED_METHOD_CONTRACT; return (dwStubFlags == ILSTUB_SECUREDELEGATE_INVOKE); }
```

Comment:
Maybe also rename SecureDelegate to something more sensible while you are on it?

Reply:
They are now WrapperDelegate, and I removed the last vestige of "creator" assembly from the data structures.